services/email/templates/caseworker/html/lost_or_stolen.mus

```diff
@@ -54,7 +54,7 @@
               <td width="75%" style="font-family: Helvetica, Arial, sans-serif;">
                 {{#data}}
 
-                  <p style="font-size: 23px; color: #000; line-height: 1.32; margin: 0 0 20px 0;">{{#t}}emails.lost-stolen.header{{/t}}</p>
+                  <p style="font-size: 23px; color: #000; line-height: 1.32; margin: 0 0 20px 0;">BRP Reported as lost or stolen</p>
```

Comment:
this was just stuff I was trying to add to the caseworker email to make it less confusing since all the title changes were for the customer. I'll lowercase the R and shove it in a translation